quantum/process_keycode/process_leader.c

```diff
@@ -49,24 +49,24 @@ bool process_leader(uint16_t keycode, keyrecord_t *record) {
   // Leader key set-up
   if (record->event.pressed) {
     if (leading) {
-      if (timer_elapsed(leader_time) < LEADER_TIMEOUT) {
-#ifndef LEADER_KEY_STRICT_KEY_PROCESSING
+    //   if (timer_elapsed(leader_time) < LEADER_TIMEOUT) {
+        #ifndef LEADER_KEY_STRICT_KEY_PROCESSING
         if ((keycode >= QK_MOD_TAP && keycode <= QK_MOD_TAP_MAX) || (keycode >= QK_LAYER_TAP && keycode <= QK_LAYER_TAP_MAX)) {
           keycode = keycode & 0xFF;
         }
-#endif // LEADER_KEY_STRICT_KEY_PROCESSING
+        #endif // LEADER_KEY_STRICT_KEY_PROCESSING
```

Comment:
Please revert this formatting change:
```suggestion
#endif // LEADER_KEY_STRICT_KEY_PROCESSING
```